src/Main/Comments.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+import { useState, useEffect } from "react"
+
+const COMMENT_TO_POST = {
+
+    senddata: ''
+}
+function Comment(props) {
+    const [commentForm, setCommentForm] = useState(COMMENT_TO_POST)
+    const { user, comments, setComments } = props
+
+
+    const postComment = () => {
+        console.log(comments)
+        console.log('.....')
+        const options = {
+            method: 'POST',
+            headers: { 'Content-Type' : 'application/json' },
+            body: JSON.stringify({
+                postId: user.id,  // Assuming user.id is the postId
```

Comment:
the logged in user (contactId = 1) is the one being able to comment, the postId is the Id of the post! – this could lead to weird malfunctions that are hard to debug, maybe take another look at this